source/application/hooks/patterns/pattern.js

```diff
@@ -379,24 +379,25 @@ export class Pattern {
 	}
 
 	toJSON() {
-		let copy = Object.assign({}, this);
-
-		for (let environmentName of Object.keys(copy.results)) {
-			let environmentResult = copy.results[environmentName];
-				for (let resultName of Object.keys(environmentResult)) {
-					let result = environmentResult[resultName];
-
-					copy.results[environmentName][resultName] = {
-						'name': resultName,
-						'source': result.source.toString('utf-8'),
-						'demoSource': result.demoSource ? result.demoSource.toString('utf-8') : '',
-						'buffer': result.buffer.toString('utf-8'),
-						'demoBuffer': result.demoBuffer ? result.demoBuffer.toString('utf-8') : '',
-						'in': result.in,
-						'out': result.out
-					};
-				}
-		}
+		let copy = {...this};
+		Object.values(copy.files).forEach(file => delete file.pattern);
```

Comment:
Why do you delete copy.files[..].pattern here if you delete copy.files down below?